editor/i18n/en/localization.js

```diff
@@ -77,11 +77,12 @@ module.exports = {
         skyIllum: 'Ambient lighting intensity.',
     },
     skybox: {
-        applyDiffuseMap: 'When checking, scene objects will use accurate diffusion map instead of hemisphere lighting.',
+        envLightingType: 'Choose environment lighting types: Custom hemisphere diffuse, no reflections / Autogen hemisphere diffuse, with reflections / More accuracy diffuse map, with reflections.',
```

Comment:
The key 'envLightingType' is not match with `@tooltip('i18n:skybox.EnvironmentLightingType')` .
![image](https://user-images.githubusercontent.com/13517177/192266634-86398423-744a-45ab-ace0-0a95073a9727.png)